### PR TITLE
Minor phpstan-fixes for php8.1

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Some fixes for php8.1 (bc)

### Related Pull Requests

1. See OpenMage/magento-lts#3163

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

File: lib/Varien/Data/Form/Element/Time.php [hunk 1]
```diff
@@ -57,7 +57,7 @@ public function getElementHtml()
 
         if ($value = $this->getValue()) {
             $values = explode(',', $value);
-            if (is_array($values) && count($values) == 3) {
+            if (count($values) == 3) {
```
Comment (kiatng) [inline lib/Varien/Data/Form/Element/Time.php:60]:
```suggestion
            if (count($values) === 3) {
```

**Review by kiatng — APPROVED** [review]

**Review by elidrissidev — DISMISSED** [review]

**Comment by elidrissidev** [comment]
I just noticed now that this is targeting `v19`, probably should've targeted `main` and then got backported.

**Review by ADDISON74 — APPROVED** [review]

**Comment by ADDISON74** [comment]
I leave this PR up to you to decide merging it in v19 then cherry picking in main.

**Comment by fballiano** [comment]
if this gets rebased to main I'll merge it right away

**Comment by sreichel** [comment]
It does not get rebased on its own.

**Comment by fballiano** [comment]
this guide by Colin fixes it https://github.com/OpenMage/rfcs/pull/10#issuecomment-1496427788

**Pull request merged**